modules/req.js

```diff
@@ -1,49 +1,129 @@
 const cliConfs = require("./cliConfs");
-const request = require("request");
+const fetch = require("node-fetch");
 
-function get(path, config) {
+function get(path, config, url = null, bool = false) {
   return new Promise((resolve, reject) => {
-    let url = cliConfs.getApiUrl() + path;
-
-    request.get({
-      headers: {
-        "x-auth-token": config.token
-      },
-      url: url,
-      json: true,
-    }, function optionalCallback(err, httpResponse, body) {
-      if (err || httpResponse.statusCode != 200) {
-        reject(body);
-      } else {
-        resolve(body);
-      }
-    });
-  });
+    if (url === null) url = cliConfs.getApiUrl() + path;
+    fetch(url, {
+        headers: {
+          "content-type": "application/json",
+          "x-auth-token": config.token,
+          'User-Agent': 'express'
+        },
+      })
+      .then(response => {
+        if (response.ok) {
+          if (!bool) {
+            response.json().then(function (data) {
+              resolve(data);
+            })
+          } else resolve(true)
+        } else {
+          if (!bool) {
+            response.json().then(function (data) {
+              reject(data);
+            })
+          } else resolve(false)
+        }
+      })
+      .catch(function (err) {
+        console.log(err)
+      })
+  })
 }
 
-function post(path, params, config) {
+function post(path, params, config, url = null) {
   return new Promise((resolve, reject) => {
-    let url = cliConfs.getApiUrl() + path;
+    if (url === null) url = cliConfs.getApiUrl() + path;
+    fetch(url, {
+        method: 'POST',
+        headers: {
+          "content-type": "application/json",
+          "x-auth-token": config.token,
+          'User-Agent': 'express'
+        },
+        body: JSON.stringify(params)
+      })
+      .then(response => {
+        if (response.ok) {
```

Comment:
this part is repeated multiple times, refactor by making a function and calling it